bindings/node.js/kzg.cxx

```diff
@@ -173,31 +224,45 @@ Napi::Value ComputeKzgProof(const Napi::CallbackInfo& info) {
   KZGProof proof;
   C_KZG_RET ret = compute_kzg_proof(
     &proof,
-    (Blob *)blob,
-    (Bytes32 *)z_bytes,
+    blob,
+    z_bytes,
     kzg_settings
   );
 
   if (ret != C_KZG_OK) {
      Napi::Error::New(env, "Failed to compute proof")
       .ThrowAsJavaScriptException();
-    return env.Undefined();
+    return env.Null();
   };
 
-  return napi_typed_array_from_bytes((uint8_t *)(&proof), BYTES_PER_PROOF, env);
+  return Napi::Buffer<uint8_t>::Copy(env, reinterpret_cast<uint8_t *>(&proof), BYTES_PER_PROOF);
 }
 
-// computeBlobKzgProof: (blob: Blob, setupHandle: SetupHandle) => KZGProof;
-Napi::Value ComputeBlobKzgProof(const Napi::CallbackInfo& info) {
-  auto env = info.Env();
 
+/**
+ * Given a blob, return the KZG proof that is used to verify it against the
+ * commitment.
+ * 
+ * @param {Blob} blob - The blob (polynomial) to generate a proof for
```

Comment:
In the C file, I think we should document all of the parameters with `[in]` or `[out]`. Most of them are already like this, but some are missing it.

Reply:
Updated.